### ISSUE TITLE
Unable to restart NVDA when in Firefox

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1635]
**Reported by elliott94 on 2011-07-01 19:42**
When in Firefox and an attempt to restart NVDA is performed (by using CTRL-Alt-N), nothing happens. What's more, even if Firefox doesn't currently have focus, the same can be seen. Here's how to reproduce:
1. Open Firefox, and set focus to any web page.
2. Attempt to use the NVDA shortcut (by default CTRL-Alt-N).
3. When nothing happens, Alt-Tab to another window, but ensure that it's not the Desktop.
4. Once again, attempt to restart NVDA.

Expected result: NVDA should restart as normal.

Actual result; nothing happens.

**Comment by dkager** [comment]
Another issue that looks abandoned.

**Comment by jcsteh** [comment]
No further info from reporter, so closing.
